blog/2024-09-23-rattler-to-conda.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: "Rattler is moving to the conda organization"
+slug: "rattler-moving-to-conda-org"
```

Comment:
Please include the date in the slug (or remove this line so the filename is used instead).